apps/prairielearn/src/pages/instructorGradebook/instructorGradebook.sql

```diff
@@ -146,23 +146,16 @@ SELECT
   uin,
   user_name,
   role,
-  ARRAY_AGG(
+  JSONB_OBJECT_AGG(
```

Comment:
This was changed from an array to an object keyed by assessment ID. This allows the system to not rely on the specific ordering of the object (particularly in the CSV download process) and to represent the data with better types, instead of custom columns.

Reply:
I have not changed the query to remove the score entries that have no assessment instance, as I didn't want to do that in this current PR, but this should be possible now, as the code that uses this column does not assume that the score for a particular assessment exists, and defaults to null values if it doesn't.

Reply:
This is now changed in 8f330941e.